techniques/css/C22.html

```diff
@@ -321,32 +318,32 @@ <h4>The CSS component</h4>
 }</code></pre>
           </section>
          
-        <p>The CSS <code class="language-css">letter-spacing</code> property is used to display the letters closer together in the second line of text.</p>
+        <p>CSS の <code class="language-css">letter-spacing</code> プロパティは、見出しでさらに離れた文字を表示するために使用される。</p>
         <section id="the-html11">
         
-          <h4>The HTML component</h4>
+          <h4>HTML コンポーネント</h4>
          
 <pre xml:space="preserve"><code class="language-html">&lt;h1 class="upper2"&gt;News&lt;/h1&gt;</code></pre>
         </section>
         <section id="the-css11">
-            <h4>The CSS component</h4>
+            <h4>CSS コンポーネント</h4>
          
 <pre xml:space="preserve"><code class="language-css">.upper2 { text-transform:uppercase; letter-spacing:1em; }</code></pre>
         </section> 
       </section>
       <section class="example" id="example-10-using-css-background-image-to-layer-text-and-images">
-         <h3>Example 10: Using CSS <code class="language-css">background-image</code> to layer text and images</h3>
+         <h3>事例 10: テキストと画像をレイヤー化するために CSS の <code class="language-css">background-image</code> を使用する</h3>
          
-				 <p>The CSS <code class="language-css">font-style</code> property is used to display the textual component of a banner and <code class="language-css">background-image</code> property is used to display a picture behind the text.</p>
+				 <p>CSS の <code class="language-css">font-style</code> <code class="language-css">background-image</code> プロパティを使って、テキストの後ろの画像を表示させる。</p>
```

Comment:
段落冒頭の「The CSS font-style property is used to display the textual component of a banner」の訳が抜けているようです。

```suggestion
				 <p>CSS の <code class="language-css">font-style</code> プロパティを使って、バナーのテキスト要素を表示させ、<code class="language-css">background-image</code> プロパティを使って、テキストの後ろの画像を表示させる。</p>
```